src/Classes/ServiceAPI/MyRadio_TracklistItem.php

```diff
@@ -187,6 +187,12 @@ public static function getTracklistForTime($start, $end = null, $include_playout
             unset($data['digitised']);
             unset($data['deletelink']);
             unset($data['trackno']);
+            unset($data['intro']);
+
+            //for manual SIS entries
+            if (!isset($data['trackid'])) {
+                $data['trackid'] = "SIS Manual";
```

Comment:
Is the fact that this isn't an integer going to have implications? perhaps -1 or 0 would be better?